min_detector_distance: 4
robots_names: ['robot1', 'robot2', 'robot3']